### PR TITLE
Implement 404 error page in the Playground project (#1891)

### LINKED ISSUE #1891: Implement the `404 Error` page in the `playground` project
The `404 Error` page must be implemented based on the new UI/UX design.

### PR DESCRIPTION
**Pull request by sajadkardel**
this closes #1891

### CODE REVIEW

**Comment by msynk** [comment]
@maryamhdr and @Cyrus-Sushiant

plz review

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/App.razor [hunk 1]
```diff
@@ -6,9 +6,7 @@
                 <RouteView RouteData="@routeData" DefaultLayout="@typeof(MainLayout)" />
             </Found>
             <NotFound>
-                <LayoutView Layout="@typeof(MainLayout)">
-                    <p>Sorry, there's nothing at this address.</p>
-                </LayoutView>
+                <PlaygroundNotFound />
```
Comment (msynk) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/App.razor:9]: shouldn't we use the main layout for the new component?
@maryamhdr

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Components/PlaygroundNotFound.razor [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿
```
Comment (msynk) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Components/PlaygroundNotFound.razor:1]: please rename this component to `NotFoundComponent`

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/compilerconfig.json [hunk 3]
```diff
@@ -1,138 +1,142 @@
...
+    "inputFile": "Pages/Components/RadioButtonGroup/BitRadioButtonGroupDemo.razor.scss"
+  },
+  {
+    "outputFile": "Pages/Components/Persona/BitPersonaDemo.razor.css",
+    "inputFile": "Pages/Components/Persona/BitPersonaDemo.razor.scss"
+  },
+  {
+    "outputFile": "Components/PlaygroundNotFound.razor.css",
+    "inputFile": "Components/PlaygroundNotFound.razor.scss"
+  }
```
Comment (msynk) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/compilerconfig.json:141]: please undo the code formatting changes.
beware of these changes and do not push them in a PR like this.

**Review by msynk — CHANGES_REQUESTED** [review]
please rename the `not-found.svg` to `404.svg`

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Components/NotFoundComponent.razor.scss [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+    width: 100%;
+    height: 100%;
+}
+
+.cloud-1 {
+    background-image: url('/images/cloud.svg');
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+    width: 88px;
```
Comment (Cyrus-Sushiant) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Components/NotFoundComponent.razor.scss:17]: I think you should use the rem function for all the places where you have entered the value in pixels.
@maryamhdr

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Components/NotFoundComponent.razor [hunk 5]
```diff
@@ -0,0 +1,12 @@
+﻿
```
Comment (Cyrus-Sushiant) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Components/NotFoundComponent.razor:1]: Plz, remove the blank line

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Components/NotFoundComponent.razor [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+<div class="main">
+    <div class="cloud-1"></div>
+    <div class="cloud-2"></div>
+    <div class="cloud-3"></div>
+    <div>
+        <div class="title"></div>
+        <div class="description">Page Not Found</div>
+    </div>
+    <div class="disconnected-cable"></div>
+    <a href="#" Class="back-btn">Go Back</a>
```
Comment (Cyrus-Sushiant) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Components/NotFoundComponent.razor:11]: I think it is better to use BitLink and set href to the website's root. What do you think? @msynk

**Review by Cyrus-Sushiant — CHANGES_REQUESTED** [review]

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Components/NotFoundComponent.razor (hunk 6 above)
Comment (msynk) [reply src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Components/NotFoundComponent.razor:10]: It's better to use the `BitButton` based on the design I saw.
and also add the functionality to go back to the home page.
@sajadkardel

**Review by Cyrus-Sushiant — APPROVED** [review]

**Review by msynk — APPROVED** [review]

**Comment by msynk** [comment]
@maryamhdr plz review

**Review by maryamhdr — APPROVED** [review]

**Pull request merged**